### PR TITLE
CONTRAST-9040

### PR DESCRIPTION
**Pull request by jleo32**
Added intregrations.md to user/teamserver/guide/integrations.

### CODE REVIEW

File: content/user/teamserver/guide/integrations/integrations.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+2. Click 'Connect' in the Slack row.
+3. Name the integration and paste the URL.
+4. Select the appropriate application(s) to filter on.
+5. Click save.
+
+You are now connected!
+
+## HipChat Integration
+
+TeamServer now supports HipChat integration! With this new connection you can receive notifications from TeamServer to your configured HipChat in a similar format to in-app notifications.
```
Comment (SteveFeldman) [inline content/user/teamserver/guide/integrations/integrations.md]: @jleo32 I would recommend we remove the word "now" and "new". Let's make it so this document has a shelf life longer than a few weeks.

File: content/user/teamserver/guide/integrations/integrations.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## Slack Integration
+
+TeamServer now supports Slack integration! With this new connection you can receive notifications from TeamServer to your configured Slack instance in a similar format to in-app notifications.
+
+## Setup
+
+To add, go to your teams 'Build' settings in Slack:
+
+1. Add a new 'Incoming Webhooks' custom integration.
```
Comment (SteveFeldman) [inline content/user/teamserver/guide/integrations/integrations.md]: Use bullets

File: content/user/teamserver/guide/integrations/integrations.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+2. Choose an appropriate channel to send messages to.
+3. Copy the 'Webhook URL'.
+
+To add in TeamServer:
+
+1. Go to the new 'Integrations' tab in Organization Settings.
```
Comment (SteveFeldman) [inline content/user/teamserver/guide/integrations/integrations.md]: Use bullets

File: content/user/teamserver/guide/integrations/integrations.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+## HipChat Integration
+
+TeamServer now supports HipChat integration! With this new connection you can receive notifications from TeamServer to your configured HipChat in a similar format to in-app notifications.
+
+## Setup
+
+In the HipChat website, login as a user with at least 'Send Notification' scope. Typically this is an admin user.
+
+1. Edit your profile and click 'API access'.
```
Comment (SteveFeldman) [inline content/user/teamserver/guide/integrations/integrations.md]: Use bullets

File: content/user/teamserver/guide/integrations/integrations.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+2. Create a new token and add at least the 'Send Notification' scope.
+3. Copy the token created.
+
+To add in TeamServer:
+
+1. Go to the new 'Integrations' tab in Organization Settings.
```
Comment (SteveFeldman) [inline content/user/teamserver/guide/integrations/integrations.md]: Use bullets

File: content/user/teamserver/guide/integrations/integrations.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+```javascript
+{
+	'title':   'Contrast Security Notification',
+	'message': 'Test User commented on a Insecure JSP Placement vulnerability in WebGoat. \"Fixed in CVE-2015\"'
+}
+```
+
+## Setup
+
+1. Retrieve the URL you want TeamServer to send notification messages to.
```
Comment (SteveFeldman) [inline content/user/teamserver/guide/integrations/integrations.md]: Use bullets

**Comment by jleo32** [comment]
@SteveFeldman Fixed all items.

**Pull request merged**